### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by ashrayjain** [jesseweed/seti-syntax#67]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `merge-conflicts` package
```
ThemePackage.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app/src/package.js:448:9)
PackageWatcher.watch (/Applications/Atom.app/Contents/Resources/app/node_modules/dev-live-reload/lib/package-watcher.js:44:35)
```

**Comment by asantos3** [comment]
[There's a open pull request for that.](https://github.com/jesseweed/seti-syntax/pull/66)

**Comment by jesseweed** [comment]
This should be fixed in v0.4
